### ISSUE TITLE
Fix download_data.py script

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#614]
2 problems.  Problem one is that `download_data.py` is failing when trying to fetch a non-existent file:
```
$ python scripts/download_data.py
Cleaning out directory '/Users/dannyc/ga4gh/server/ga4gh-downloaded-data/dataset1/variants/1kg-phase3-subset'
Downloading 'ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20130502/ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz'
Writing 'ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.unsampled'
Sampling 'ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.unsampled'
Compressing 'ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf'
Indexing 'ALL.chr1.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz'
chrom: 1, maxPos: 138395, minPos: 10176
Downloading 'ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20130502/ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz'
Writing 'ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.unsampled'
Sampling 'ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.unsampled'
Compressing 'ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf'
Indexing 'ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz'
chrom: 2, maxPos: 32403, minPos: 10178
Downloading 'ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20130502/ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz'
Writing 'ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.unsampled'
Sampling 'ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.unsampled'
Compressing 'ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf'
Indexing 'ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz'
chrom: 3, maxPos: 81617, minPos: 60068
Downloading 'ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20130502/ALL.chrX.phase3_shapeit2_mvncall_integrated_v1a.20130502.genotypes.vcf.gz'
Traceback (most recent call last):
  File "scripts/download_data.py", line 354, in <module>
    main(args)
  File "/Users/dannyc/ga4gh/server/scripts/utils.py", line 34, in wrapper
    result = func(*args, **kwargs)
  File "scripts/download_data.py", line 347, in main
    downloader.downloadVcfs()
  File "scripts/download_data.py", line 226, in downloadVcfs
    self._processFileName(fileName)
  File "scripts/download_data.py", line 189, in _processFileName
    response = urllib2.urlopen(url)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1401, in ftp_open
    fp, retrlen = fw.retrfile(file, type)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib.py", line 903, in retrfile
    self.ftp.cwd(file)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ftplib.py", line 553, in cwd
    return self.voidcmd(cmd)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ftplib.py", line 249, in voidcmd
    return self.voidresp()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ftplib.py", line 224, in voidresp
    resp = self.getresp()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ftplib.py", line 219, in getresp
    raise error_perm, resp
urllib2.URLError: <urlopen error ftp error: [Errno ftp error] 550 ALL.chrX.phase3_shapeit2_mvncall_integrated_v1a.20130502.genotypes.vcf.gz: No such file or directory>
```
Problem two is that the directory structure needs to be changed to reflect the changes in #605

**Comment by jeromekelleher** [comment]
I would deal with problem 1 by removing support for X and Y. They seem to have changed files, and they don't have samples for the individuals we have so let's just get rid of them.

**Comment by dcolligan** [comment]
Fixed in #616
